tracing/tracers/lightstep/README.md

```diff
@@ -0,0 +1,24 @@
+# lightstep tracer
```

Comment:
Can you please add a link from https://opensource.zalando.com/skipper/operation/operation/#opentracing (docs/operation/operation.md) to here? 
thanks!

Reply:
linked to master where it will appear when it's merged